_posts/2020/2020-01-20-be-suspicious-of-join-tables.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Be Suspicious of Join Tables"
+description: "They're probably hiding a concept you haven’t named yet"
+layout: article
+category: ruby
+image:
+  base: '2020/be-suspicious-of-join-tables'
+  alt: Welding
+  credit:  Max LaRochelle
+  source: "https://unsplash.com/photos/QzP1GcDOSC8"
+
+---
+
+We often have to represent many-to-many relationships between models in our applications. Rails provides a method in the migrations to generate a table in your database to support this. You can see the documentation in the [Rails guide for ActiveRecord migrations](https://guides.rubyonrails.org/active_record_migrations.html#creating-a-join-table).
+
+However these “pure” join tables often obscure a useful concept in your application that might be better represented as a named model.
```

Comment:
* comma after "However".

Not sure about use of 'pure' here. Not sure what meaning it's trying to convey.